php/libraries/NDB_Config.class.inc

```diff
@@ -450,7 +450,11 @@ class NDB_Config
                 array('sp' => $subprojectID)
             );
         } catch(DatabaseException $e) {
-            return null;
+            throw new LorisException(
+                'While attempting to get Subproject config settings, caught ' .
+                "DatabaseException: `$e`"
+            );
+            return array();
```

Comment:
This return is unreachable.. but other than that, what's the point of this change?

Reply:
I'll remove the unreachable return. 

I thought it would be better not to silence the DB exception because it probably means something went wrong. Returning `null` to me makes it seem like there was no entry for `$subprojectID` in the table -- but in that case there shouldn't be an exception thrown; `pselectRow` should just return null.

Reply:
In that case why not just remove the try/catch? If the database throws an exception and it isn't caught the stack trace will tell you where it came from.